.github/workflows/macOS-tests.yml

```diff
@@ -39,6 +39,10 @@ jobs:
           pip3 install --upgrade pip
           pip3 install codespell
           codespell --count --ignore-words-list=ans,deriver,inout,packag --skip="*.js,*WordLists.swift"
+      - name: SwiftLint
+        run: |
+          swiftlint --fix
+          # swiftlint analyze
```

Comment:
Please, remove `# swiftlint analyze` or maybe leave a comment about why we may need it later.